portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +94,36 @@ function closeForm() {
 }
 
 
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
 
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
 
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("demo");
```

Comment:
This mixes your art and projects together which can be a little confusing because when I was clicking on your slideshow I was only expecting art. I'm not sure if that was your intention.